perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh

```diff
@@ -28,7 +28,7 @@ commitId=$(git log --before='yesterday 23:59:59' --max-count=1 --pretty=%H)
 git checkout $commitId
 
 echo "Executing integration tests"
-GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/... -p 1 --integrationTest -v --testbucket=gcsfuse-integration-test
+GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/... -p 1 --integrationTest -v --testbucket=gcsfuse-integration-test -timeout=20m
```

Comment:
Is building gcsfuse taking most of time in running these tests ?

Reply:
These tests in [operations](https://github.com/GoogleCloudPlatform/gcsfuse/tree/master/tools/integration_tests/operations) are taking time to run